scripts/update_data_propub

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+source .env.secrets
+
+if [[ -z "$PROPUBLICA_API_KEY" ]]; then
+    echo "Error. Must ensure /scripts/.env.secrets file exists and contains your PROPUBLICA_API_KEY. See /scripts/.env.secrets.example for more details." 1>&2
+    exit 1
+fi
+
+echo "ProPublica API key found. Updating Congress (House + Senate) data...."
+
+
+curl -H "X-API-Key: ${PROPUBLICA_API_KEY}" "https://api.propublica.org/congress/v1/117/senate/members.json" > ../data/cawp/propublica-us-senate.json
```

Comment:
Can you throw these into a `main` function. Check out the other scripts for an example on how to do that.